.travis.yml

```diff
@@ -8,8 +8,8 @@ addons:
    repo_token: 4d2c1cec2a5ba5fd0cd09aa76d1bcb52854e12ace21660dbf65a36a59ba7a973
 
 rvm:
-  - 2.0.0
```

Comment:
We still need to support ruby 2.0 in CocoaPods until Xcode 10 ships.

Reply:
Thank you for your reviewing.
I'm going to manage to do so.

Reply:
Thank you for your reviewing.
Perhaps I have to wait until this issue resolved... https://github.com/travis-ci/travis-ci/issues/9190 .